hw/dv/tools/dvsim/vcs.hjson

```diff
@@ -42,7 +42,45 @@
                //  - Read capability on registers, variables, and nets
                //  - Write (deposit) capability on registers and variables
                //  - Force capability on registers, variables, and nets
-               "-debug_access+f"]
+               "-debug_access+f",
+
+               // Upgrade below warnings to errors to make VCS more strict on syntax to avoid
+               // having issue with other simulatior but passing with VCS
```

Comment:
Nit: Typo in "simulator" (and I think it should be "simulators" anyway).

Reply:
Good catch, updated. Thanks